stable/dask/Chart.yaml

```diff
@@ -1,10 +1,12 @@
-name: dask-distributed
-version: 2.0.0
+name: dask
+version: 1.0.0
```

Comment:
What's the reason for this? You rename the chart and decrease the version? Seems weird.

Reply:
Happy to increase the version if desired.  I decreased the version to 1.0 because of the name change.  I'm happy to adhere to any standard though.